### DISCUSSION TITLE
Changing ESRB raiting for a game

### DISCUSSION THREAD

**Discussion by ivshof** [d0k3/GodMode9#698] [Q&A]
I am impressed by the variety of features GodMode9 provides. However, I wonder if we can change ESRB rating for games.

The problem is that Minecraft on my NewNintendo3ds keeps asking for parental ctrl PIN... and I already allowed all game ratings in 3ds parental manager...

I wonder if we can change those requirements for a game, or before installation for CIA| 3DS file, so particular games won't ask that PIN.
Have no idea why some kids game (Minecraft seems to be 6+) asking for parental PIN... could be something with other game feature not only ESRB, or something in CIA file I guess.

Sorry, if I asked in the wrong place.

**Comment by wg481** [comment]
This happened to me too!

Apparently, Parental Controls liked to disable content installed through FBI/GM9. Just straight up turn off PControls and it should work again.
